chunky/src/java/se/llbit/util/TaskTracker.java

```diff
@@ -114,6 +113,45 @@ public void update(String task, int target, int done, String eta) {
       this.eta = eta;
       update();
     }
+
+    /** Changes the task name and state. Calculates eta. */
+    public void update(String task, int target, int done, long startTime) {
+      long etaSeconds = 0;
+      if (done > 0) {
```

Comment:
The ETA should only be shown if a start time was set.

Reply:
Updating with the ETA is now under `updateEta`. The ETA start time is pulled from the task's `startTime` which is set to the task creation time, or when `Task.updateStartTime` is called.